### DISCUSSION TITLE
C++ in zed

### DISCUSSION THREAD

**Discussion by Krishnav1237** [zed-industries/zed#8962] [Feature Requests]
where to check the output of a c++ programme in zed , i am unable to see the output and the build.

**Comment by ghost** [comment]
You can build and run your code in the built-in terminal (`CMD-j` is the default keybinding to pop up the terminal)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
